runtime/doc/nvim_clipboard.txt

```diff
@@ -29,4 +35,77 @@ following option:
 See 'clipboard' for details and more options.
 
 ==============================================================================
+2. X11 selection mechanism		      *nvim-clipboard-x11* *x11-selection*
+
+X11 provides two basic types of global store, selections and cut-buffers,
```

Comment:
I don't think this should mention cut-buffers so prominently, and give reader (unfamiliar with the inner workings of X11) the impression that they are on equal footing with selections. They are generally considered obsolete (as they don't support formats, for instance) and many modern applications simply don't use them at all.